src/insertItem.ts

```diff
@@ -43,43 +42,22 @@ export function getItemType(): QuickPickItem<ItemType>[] {
     return items;
 }
 
-function getResourceSnippets(): vscode.QuickPickItem[] {
-    let items: vscode.QuickPickItem[] = [];
-    let snippetPath = path.join(assetsPath, "armsnippets.jsonc");
-    let content = fse.readFileSync(snippetPath, "utf8");
-    let tree = Json.parse(content);
-    if (!(tree.value instanceof Json.ObjectValue)) {
-        return items;
-    }
-    for (const property of tree.value.properties) {
-        if (isResourceSnippet(property)) {
-            items.push(getQuickPickItem(property.nameValue.unquotedValue));
+async function getResourceSnippets(): Promise<IAzureQuickPickItem<ISnippet>[]> {
```

Comment:
This code is for "insert item", which looks at our snippets which are ARM resources to populate its list, then calls vscode to insert the resource that the user selects.  Now instead goes through the snippet manager